### ISSUE TITLE
Uncaught TypeError: Cannot read property 'tokenizePlaceholders' of undefined

### ISSUE THREAD

**Issue by jwillmer** [PrismJS/prism#1423]
I updated prism to `1.14.0` because I needed to support another language and since then I get the following error on my page:

```
prism.js:formatted:1119 Uncaught TypeError: Cannot read property 'tokenizePlaceholders' of undefined
    at prism.js:formatted:1119
    at Object.run (prism.js:formatted:249)
    at Object.highlight (prism.js:formatted:154)
    at Object.highlightElement (prism.js:formatted:139)
    at Object.highlightAllUnder (prism.js:formatted:97)
    at HTMLDocument.highlightAll (prism.js:formatted:88)
```

If I follow `prism.jsformatted:1119` I get to the following code:

```
e.hooks.add("after-tokenize", function(n) {
   e.languages["markup-templating"].tokenizePlaceholders(n, "php")
})
```

**Comment by Golmote** [comment]
You need to include the `markup-templating` component in your code. See related #1395 and #1400.

**Comment by jwillmer** [comment]
What is `markup-templating`? The website says I only need to add the `js` and `css` to my website in order to work. Is the website wrong or is the current version broken?

**Comment by jwillmer** [comment]
I use the following link to get the css and js:
http://prismjs.com/download.html?#themes=prism&languages=markup+css+clike+javascript+c+csharp+bash+batch+cpp+aspnet+arduino+coffeescript+ruby+git+java+json+less+objectivec+php+powershell+python+scss&plugins=line-highlight+line-numbers

I implement it in a static web site by referencing the css and js file.

**Comment by Golmote** [comment]
It should be fixed with c2ff24822a750f2aad24b9393bd78c6e896fe633. Can you please try to redownload Prism using the same link you posted above? You should see that the checkbox "Markup templating" is automatically checked now.

**Comment by Volmarg** [comment]
In my case I had this
`import Prism from 'prismjs';`

should've been:

`import * as Prism from 'prismjs';`
